scss/_variables.scss

```diff
@@ -447,14 +447,18 @@ $custom-file-border-radius: $input-border-radius !default;
 $custom-file-box-shadow:    $input-box-shadow !default;
 $custom-file-button-color:  $custom-file-color !default;
 $custom-file-button-bg:     $input-group-addon-bg !default;
-$custom-file-text: (
+
+// stylelint-disable
```

Comment:
Why do you disable stylelint here exactly?

Reply:
Stylelint suggests that maps begin on separate lines. I could have put the parentheses in separate lines but as in the color variables it is disabled and so it was approved, I thought it was the right way. It can be re-enabled but to pass the test have to put all parentheses in separate lines. And it should also be done in color variables.

Reply:
Ok, thanks for the explanation. There must be some way to tweak the rule for this, but I didn't have the time to look at it.

Either way, can you just disable the specific stylelint error instead of all?

```
// stylelint-disable rule
...
//stylelint-enable rule
```

Reply:
Yes, of course, I do now.